components/ui/Anchor.jsx

```diff
@@ -9,9 +9,9 @@ export default function Anchor({ href, children, style }) {
       href={href}
       className="inline-flex gap-1  md:leading-relaxed leading-loose font-normal md:text-2xl text-xl tracking-normal md:tracking-wide group"
     >
-      <span className={`relative after:absolute after:bottom-0.5 after:left-0.5 after:right-0 after:w-auto after:h-[1px] after:bg-black inline-block ${style}`}>
+      <span className={`relative after:absolute after:bottom-0.5 after:left-0.5 after:right-0 after:w-auto after:h-[1px] group-hover:after:bg-white after:bg-black inline-block ${style}`}>
```

Comment:
It would be better if we turned the bottom border transparent rather than white so that if we have a background that is not white, it still disappears